README.md

```diff
@@ -87,6 +87,7 @@ $ make docker/start
 Any wallet that supports WalletConnect 2.0 should work. Specifically, we've tested:
```

Comment:
This is not correct then. There are non-WC wallets. Please also mention OneGate.